rewritingTask2/tests.c

```diff
@@ -0,0 +1,59 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include "insertionSort.h"
+
+bool arraysEqual(int arr1[], int arr2[], int size) {
+    for (int i = 0; i < size; i++) {
+        if (arr1[i] != arr2[i]) {
+            return false;
+        }
+    }
+    return true;
+}
+
+bool test1() {
```

Comment:
```suggestion
bool test1(void) {
```
Если у функции не указаны аргументы, считается, что она принимает произвольное число аргументов, так что компилятор не выдаст ошибку, например, на test1(1, 2, 3). А мы хотим, чтобы в таких случаях выдавал.